pkg/controllers/management/drivers/kontainerdriver/kontainerdriver.go

```diff
@@ -132,6 +132,58 @@ func (l *Lifecycle) download(obj *v3.KontainerDriver) (*v3.KontainerDriver, erro
 }
 
 func (l *Lifecycle) createDynamicSchema(obj *v3.KontainerDriver) error {
+	resourceFields, err := l.getResourceFields(obj)
+	if err != nil {
+		return err
+	}
+
+	dynamicSchema := &v3.DynamicSchema{
+		Spec: v3.DynamicSchemaSpec{
+			SchemaName:     getDynamicTypeName(obj),
+			ResourceFields: resourceFields,
+		},
+	}
+	dynamicSchema.Name = strings.ToLower(getDynamicTypeName(obj))
+	dynamicSchema.OwnerReferences = []v13.OwnerReference{
+		{
+			UID:        obj.UID,
+			Kind:       obj.Kind,
+			APIVersion: obj.APIVersion,
+			Name:       obj.Name,
+		},
+	}
+	dynamicSchema.Labels = map[string]string{}
+	dynamicSchema.Labels[obj.Name] = obj.Status.DisplayName
+	dynamicSchema.Labels = map[string]string{}
+	dynamicSchema.Labels[driverNameLabel] = obj.Status.DisplayName
+	_, err = l.dynamicSchemas.Create(dynamicSchema)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("error creating dynamic schema: %v", err)
```

Comment:
Maybe preferable to use errors.WithMessage?

Reply:
Fixed